packages/core/test/render3/component_spec.ts

```diff
@@ -152,6 +152,65 @@ describe('component', () => {
 
 });
 
+it('should not invoke renderer destroy method for child views', () => {
+  let comp: Comp;
+
+  function MyComponent_div_Template_2(rf: any, ctx: any) {
+    if (rf & 1) {
```

Comment:
Nit: For tests, can we use the enum version of this?

```suggestion
    if (rf & RenderFlags.Create) {
```